src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,111 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.*;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String input = Console.readLine();
+        Map<String, Integer> race = new HashMap<>();
+        try {
+            getNames(race, input);
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(e.getMessage());
+        }
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        int num;
+        try {
+            num = parseNumber(Console.readLine());
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(e.getMessage());
+        }
+
+        System.out.println("실행 결과");
+
+        for (int i = 0; i < num; i++) {
+            for(Map.Entry<String, Integer> entry : race.entrySet()){
+                int r = randomNum();
+                move(entry, goStop(r));
```

Comment:
```suggestion
                move(entry, randomNum());
```

이렇게 작성하고, `4 이상이면 움직인다`의 로직을 `move()` 내부로 옮겨보시면 어떨까요?